kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/asset/AssetConfiguration.java

```diff
@@ -122,7 +117,7 @@ public AssetConfiguration(final String description, final String driverPid, fina
      *
      * @return the asset channels
      */
-    public Map<Long, Channel> getAssetChannels() {
+    public Map<String, Channel> getAssetChannels() {
         return this.assetChannels;
```

Comment:
It's my mistake actually 😜. Could you kindly return an unmodifiable view of the map?